exercises/two-fer/description.md

```diff
@@ -3,10 +3,10 @@
 Given a name, return a string with the message:
 
 ```text
-One for X, one for me.
+One for [name], one for me.
 ```
 
-Where X is the given name.
+Where "name" is the given name.
```

Comment:
Almost guarantee you you're going to see submissions that return 'One for "name", one for me.'.

Reply:
We'll see.  I'm not worried.

Reply:
More people use X than do some other weird type of misunderstanding...